### ISSUE TITLE
Community Patch Updates needed since Starfield 1.10.31 & 1.10.32

### ISSUE THREAD

**Issue by Aurelianis** [Starfield-Community-Patch/Starfield-Community-Patch#924]
Fixes that are no longer needed or need to be modified for better compatibility.

### Details

https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/370
https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/553
https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/369
MQ401 "New Game Plus Standard Handling" [QUST:001DC45E]
SFCP adds properties for the Coe Estate Door so that it would be unlocked for Sam's Commitment quest. This fix is no longer needed due to 1.10.31 fixing this issue within Sam's Commitment quest. UPDATE, see 1st comment below. Beth's "fix" didn't actually fix anything, but it is easy enough to fix their fix.... 
Remove Coe Door properties from 001DC45E and the modified script Fragments:Quests:QF_MQ401_001DC45E. 

SFCP also uses MQ401 "New Game Plus Standard Handling" [QUST:001DC45E] to fix #369. I suggest starting COM_Companion_SamCoe_CoraCoe_Handler "Cora Coe" [QUST:00187BF1] from the Update script instead of using MQ401 "New Game Plus Standard Handling" [QUST:001DC45E] properties and scripts to start it to maximize compatibility (this fix for Cora's quest not starting in NG+ when skipping the main quest is still needed as it is started within The Empty Nest quest which does not run when skipping the main quest). This fix is pretty important and wouldn't want it to get lost by the quest header being overwritten by another mod. The update script could listen for a setstage on MQ401 and start it at stage 450 or 455 of MQ401 (it's one or the other so would need to be set to start on either one) so that Cora isn't hanging out in the basement before you ever meet Sam in NG+.

My scripting skills are not great, but maybe something like this (untested and probably needs different conditioning to ensure it only listens if in NG+):
```
Event OnStageSet(int auiStageID, int auiItemID)
  Quest CREW_EliteCrewCoraCoe = Game.GetForm(0x00187BF1) as Quest
  Quest MQ401 = Game.GetForm(0x001DC45E) as Quest
    if (auiStageID == 450 || auiStageID == 455) && !CREW_EliteCrewCoraCoe.IsRunning()
      SFCPUtil.WriteLog("Starting Cora Coe crew quest")
      CREW_EliteCrewCoraCoe.Start()
      UnregisterForRemoteEvent(MQ401, "OnStageSet")
    EndIf   
endEvent    

  ; Fix for https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/369
    if (!b001CoraCoeFix || (CurrentVersionGTE(0,0,1)))
        Quest CREW_EliteCrewCoraCoe = Game.GetForm(0x00187BF1) as Quest
        Quest MQ401 = Game.GetForm(0x001DC45E) as Quest
        ; Added MQ401.GetStageDone(300) && (MQ401.IsStageDone(450) || MQ401.IsStageDone(455))
        ; MQ401.IsStageDone(300) makes sure that the main quest was skipped since this is the only time that the quest needs to be started
        ; (MQ401.GetStageDone(450) || MQ401.GetStageDone(455) makes sure that you have met Sam and Cora isn't just hanging out in the basement before meeting him.
        if (iTimesEnteredUnity > 0 && !CREW_EliteCrewCoraCoe.IsRunning()) && MQ401.IsStageDone(300) && (MQ401.IsStageDone(450) || MQ401.IsStageDone(455))
            SFCPUtil.WriteLog("Starting Cora Coe crew quest")
            CREW_EliteCrewCoraCoe.Start()
        else 
            RegisterForRemoteEvent(MQ401, "OnStageSet")
            SFCPUtil.WriteLog("Cora Coe crew quest does not need to be manually started. Skipping Fix.")
        endif
        b001CoraCoeFix = True
    endif
```

https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/373
This isn't anything to do with the Starfield update, but a change that I feel would be beneficial for compatibility.
This should actually be set on an INFO topic (the properties for the topic were not set properly so this is why it was not being set). This will improve compatibility as quest headers and scripts would still be vanilla, plus is appears this is how it should have been set. 
Remove COM_SamCoe_Story_SG02 [SCEN:0026C506] and the modified script Fragments:Scenes:SF_COM_SamCoe_Story_SG02_0026C506 from the Community patch and instead change the properties on
[INFO:0026C5A1] ('And if she just learned to pick up after herself she just might live until she's 13.' in GRUP Topic Children of [DIAL:0026C5E3]). Tested and verified working.

![image](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/assets/109992824/7f056a24-9b1e-4579-847d-dcc9133e1807)

https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/372
This isn't anything to do with the Starfield update, but a change that I feel would be beneficial for compatibility.
Remove COM_Quest_SamCoe_Commitment "Commitment: Sam Coe" [QUST:000DF7AD] and the modified Fragments:Quests:QF_COM_Quest_SamCoe_Commitme_000DF7AD script from the Community patch and instead change the properties on [INFO:0023DFA5] ('Well, apparently we're having a ceremony! Any time you're ready, head over to Akila City.' in GRUP Topic Children of [DIAL:0023E003]). This will improve compatibility as quest headers and scripts would still be vanilla, plus is appears this is how it should have been set. Tested and verified working.

![image](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/assets/109992824/aef066e6-0494-4ed6-93a0-fc2351910b55)
https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/885
SFCP modifies the ownership of 0022F4A2 to SyndicateFaction [FACT:00270274]. This was fixed in Starfield v1.10.31 and is no longer needed in the Community Patch.
Remove 0022F4A2 from SFCP.

https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/679
Starfield v1.10.31 adds a condition to [INFO:00187B59] ('*Laughs* Okay. Good one, Cora. Next time you see Barrett try it out on him.' in GRUP Topic Children of [DIAL:00187BC7]) so that if Barrett is dead it won't fire.
Remove COM_CoraCoe_RC_07 [QUST:00187BF5] from the Community Patch.

https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/631
This was fixed in 1.10.31.  Tested and verified working as of 1.10.31.
Remove lc116world "Nishina Research Station" [WRLD:0023B8C3] and all sub records from SFCP

### Game Version

1.10.32

### Language

English (EN)

### Have you recreated this issue on a new game?

Yes, I can recreate this issue.

### Reproduction Steps

Nothing to reproduce. Just check changes made in Starfield.esm and the changelogs.

**Comment by Aurelianis** [comment]
**UPDATE:**
The issue with the Coe Estate Door being locked during Commitment: Sam Coe when skipping the main quest is still present in v1.10.32.

First issue is that the Alias points to the wrong door.
COM_Quest_SamCoe_Commitment "Commitment: Sam Coe" [QUST:000DF7AD]
010 CoeMainDoor alias points to CoeEstateFrontDoorREF [REFR:002FC83E] but it should point to [REFR:000E69EC] instead as 000E69EC is actually the front door regardless of what the EDID says for 002FC83E

![image](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/assets/109992824/20878c5e-056a-4bf2-b72e-d2cfd023d20f)

Second issue, it still didn't unlock. Working on figuring this one out. Will update when I find a fix.

**Comment by Constellation-VASCO** [comment]
This issue has been fixed by Bethesda, and the fix in the Community Patch actually reintroduces the bug now: https://www.starfieldpatch.dev/issues/32

These records should be removed: [000A91FE] [000A920E]

**Comment by Aurelianis** [comment]
Power From Beyond quests have been improved. Since 1.10.32 I have not had any issues with missing temples and quests not starting (maybe I've just been lucky). However, I had to revert some changes made by SFCP and forward differences.

New property for MQ_Temple_SubScript added in 1.10.31 "AnomalyLocation" needs to be forwarded to the records in SFCP
000854E8
0001C179
0001C078
0001C077
0001C076
0001C075

The script fragments have been changed from "Fragments:Quests:QF_MQ_TempleQuest_Freefor_<formID>_**13**" to "Fragments:Quests:QF_MQ_TempleQuest_Freefor_<formID>_**14**" and the new fragment script name needs to be forwarded:
0001C078
0001C077
0001C076
0001C075

Stage 60 - Reward List has been modified and needs to be changed XPRewardRadiantQuestLargeCOPY0009 [GLOB:002EDFA6]
SFCP's entry is now an error as it cannot be found
0001C078
0001C077
0001C076
0001C075

TempleStarborn alias Package is now an error in SFCP and needs to be changed to MQ_TempleQuest_Freeform_StarbornPkgDUPLICATE008 [PACK:002EDFAB]
0001C078
0001C077
0001C076
0001C075

Objectives modified by SFCP seem to be still needed as well as Alias_PlanetExploredTrait being changed to None since the alias doesn't exist.

**Comment by Aurelianis** [comment]
Needs records forwarded from Starfield.esm. I won't list everything, but basically just need to open in xEdit and forward changes made in 1.10.31/32. I've tried to cross-reference the Issues list with the quests, but some I couldn't find so I'm not entirely sure if the changes are due to SFCP fixing something, or due to a change made in the latest Starfield version.

000192D2 - [508/0] Fragments:Quests:QF_CF02_000192D2:Fragment_Stage_0508_Item_00 AND Stage 508 need to be forwarded.
002A567D - Script ChangeLocationStages needs RequireExactLocation property forwarded. 
0018E0F9 - Just forward entire Quest header. This is only in SFCP to fix a dialogue spelling error as far as I can tell. #339
00003448 - Forward script fragment [155/2] Fragments:Quests:QF_MQ101_00003448:Fragment_Stage_0155_Item_02. This only appears to be in SFCP for German string issues in 2 topic infos. #345
002C1C9B - Ryleigh alias needs Allow Dead flag added. This only appears to be in SFCP for German string issues in 1 topic info. #345
0024EF9C - I couldn't find any reason for this to be in SFCP, but a script for alias 050 PlayerShipArmillaryScreenTrigger needs to be added. Objective 70 needs conditions and target forwarded. 
001DC45E - I touched on this in the original post, but if you decide to still use this quest to start Cora's quest, then the property COM_PreventStoryGateScenes for Fragments:Quests:QF_MQ401_001DC45E needs to be forwarded. #369
001FB85C - Lots of changes on this one so I won't list them all here, but this only appears to be in SFCP to mark an alias as Optional #615

**Comment by Constellation-VASCO** [comment]
That's quite a lot of changes. I'll see what I can whip up.

**Comment by Pickysaurus** [comment]
- [x] https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/885
SFCP modifies the ownership of 0022F4A2 to SyndicateFaction [FACT:00270274]. This was fixed in Starfield v1.10.31 and is no longer needed in the Community Patch.
Remove 0022F4A2 from SFCP.

- [x] https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/679
Starfield v1.10.31 adds a condition to [INFO:00187B59] ('Laughs Okay. Good one, Cora. Next time you see Barrett try it out on him.' in GRUP Topic Children of [DIAL:00187BC7]) so that if Barrett is dead it won't fire.
Remove COM_CoraCoe_RC_07 [QUST:00187BF5] from the Community Patch.

- [x] https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/631
This was fixed in 1.10.31. Tested and verified working as of 1.10.31.
Remove lc116world "Nishina Research Station" [WRLD:0023B8C3] and all sub records from SFCP

- [x] https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/32
This issue has been fixed by Bethesda, and the fix in the Community Patch actually reintroduces the bug now: 
These records should be removed: [000A91FE] [000A920E]

- [x] https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/issues/860
The following fix should be removed. It seems that it causes the game to crash upon saving changes to the ship. Unsure exactly why though, as all it does is change the faction keyword associated with the ship
